gui-apps/nwg-panel/nwg-panel-0.4.3.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Cross platform desktop notifications for Python scripts and applications."
+HOMEPAGE="https://github.com/nwg-piotr/nwg-panel/"
+SRC_URI="https://github.com/nwg-piotr/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}-src.tar.gz"
+LICENSE="MIT"
+
+SLOT="0"
+KEYWORDS="~amd64"
+DEPEND="
+	$(python_gen_cond_dep 'dev-python/pygobject[${PYTHON_USEDEP}]')
+	$(python_gen_cond_dep 'dev-python/i3ipc[${PYTHON_USEDEP}]')
+	gui-libs/gtk-layer-shell
+"
+
+src_install() {
+	insinto /usr/share/pixmaps
+	doins nwg-panel.svg
+	doins nwg-shell.svg
+	insinto /usr/share/applications
+	doins nwg-panel-config.desktop
+	python_install_all
```

Comment:
Rename this to `distutils-r1_python_install_all`.   
The `python_install_all` is an inherited function and should not be used unless inheriting the `python-r1` eclass directly.

Reply:
Doing so wouldn't install the package correctly anymore. I followed the distutils-r1 api guide [here](https://dev.gentoo.org/~mgorny/python-guide/distutils.html) which suggests that the proper way to do this is to override python_install and call distutils-r1_python_install from within that, and add custom behavior after that call.
I was overriding the doc install functions in the first place which is even less correct.

Reply:
Ah, ok. This sounds fine then :+1: